### PR TITLE
feat(crons): Add cron monitor seat to data categories

### PR DESCRIPTION
**Pull request by davidenwang**
Add monitor seat as a data category in preparation for the new checkout flow which will include cron monitors

### CODE REVIEW

File: static/app/constants/index.tsx [hunk 1]
```diff
@@ -309,6 +309,14 @@ export const DATA_CATEGORY_INFO = {
     titleName: t('Monitor Check-Ins'),
     uid: 10,
   },
+  [DataCategoryExact.MONITOR_SEAT]: {
```
Comment (davidenwang) [inline static/app/constants/index.tsx:312]: derived from https://github.com/getsentry/getsentry/blob/e657f445c1facd71fcee48682cf4e0ed6dd73b24/getsentry/billing/utils/data_category.py#L170, but some of these usages/relationships are a bit unclear lmk if something doesn't make sense here

File: static/app/constants/index.tsx [hunk 2]
```diff
@@ -309,6 +309,14 @@ export const DATA_CATEGORY_INFO = {
...
+    name: DataCategoryExact.MONITOR_SEAT,
+    apiName: 'monitorSeat',
+    plural: 'monitorSeats',
```
Comment (evanpurkhiser) [inline static/app/constants/index.tsx:315]: I think thi sshould be `monitor seats`

Comment (evanpurkhiser) [reply static/app/constants/index.tsx:315]: or maybe `cron monitors`

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/constants/index.tsx (hunk 2 above)
Comment (mbauer404) [reply static/app/constants/index.tsx:315]: I believe we want "cron monitor(s)" for anything that gets displayed in the UI

Comment (davidenwang) [reply static/app/constants/index.tsx:315]: ahh I talked with tommy/vu and I think we're changing it to just Monitors to keep it consistent with other items in the checkout flow

Comment (davidenwang) [reply static/app/constants/index.tsx]: going to change this to cron monitors and merge it in, will change to monitors on BE/FE if needed after syncing more up with Tommy

**Comment by davidenwang** [comment]
Okay after further investigation I changed the plural name back to `monitorSeats` as that is what is returned from the api and will be used in the frontend to identify which category is being rendered. The titleName was changed from `Monitors` to `Cron Monitors`

**Review by mbauer404 — APPROVED** [review]

**Pull request merged**